build.gradle

```diff
@@ -37,7 +37,7 @@ plugins {
     id 'nebula.ospackage' version "8.3.0" apply false
     id 'idea'
     id 'jacoco'
-    id 'checkstyle'
+    id "com.diffplug.spotless" version "6.3.0" apply false
```

Comment:
do we need to align with OpenSearch in versions? They are [using 6.2](https://github.com/opensearch-project/OpenSearch/blob/main/build.gradle#L51), not sure if there is much of a difference.

Reply:
agree, thanks for clarifying